integration-tests/wallet-failwith.spec.ts

```diff
@@ -3,13 +3,13 @@ import { failwithContractCode } from "./data/failwith";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Test contract made with wallet API that throws FAILWITH api using: ${rpc}`, () => {
+  describe(`Test contract origination that throws FAILWITH api through wallet api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('captures a FAILWITH and throws an error', async (done) => {
+    it('Verify wallet.originate a contract with FAILWITH code will fail and throw an error', async (done) => {
```

Comment:
It is not the origination but calling the default method that fails

Reply:
changed to
"Verify calling the default method of a contract with FAILWITH code will fail and throw an error"